gitautosync/pull_from_remote.py

```diff
@@ -0,0 +1,76 @@
+import os
+import subprocess
+
+
+def pull_from_remote(repo_name, branch_name, sync_path, account, domain):
+    """
+    Pull selected repo from a remote git repository,
+    while preserving user changes
+    """
+    assert repo_name and branch_name and account and domain
+
+    print('Starting pull.')
```

Comment:
Use the logging module here, rather than print statements. That allows us to tune how much logging output we want later - so we can pass a '--debug' flag in to set the loglevel to 'DEBUG' and actually show statements, and it can be silent during normal execution